simulator/cmd/simulator/simulator.go

```diff
@@ -11,15 +11,25 @@ import (
 	"golang.org/x/xerrors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/discovery"
+	"k8s.io/client-go/discovery/cached/memory"
+	"k8s.io/client-go/dynamic"
 	clientset "k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/rest"
+	"k8s.io/client-go/restmapper"
 	"k8s.io/klog/v2"
 
 	"sigs.k8s.io/kube-scheduler-simulator/simulator/config"
 	"sigs.k8s.io/kube-scheduler-simulator/simulator/server"
 	"sigs.k8s.io/kube-scheduler-simulator/simulator/server/di"
 )
 
+const (
+	kubeAPIServerPollInterval = 5 * time.Second
+	kubeAPIServerReadyTimeout = 2 * time.Minute
+	importTimeout             = 2 * time.Minute
```

Comment:
It would be better to make this duration configurable via `config.yaml` because the required duration varies for the source environment.

Reply:
Definitely. But I think another PR should fix it because the difference of this PR is already large. I added the issue: https://github.com/kubernetes-sigs/kube-scheduler-simulator/issues/369